tools/find_image.rb

```diff
@@ -30,17 +30,17 @@ def run
       c.action do |args, options|
         options.default \
           :build_type => "nightly-",
```

Comment:
why is there a `-` after `nightly` ? just out of curiosity.

Reply:
@kedark3 don't really rememeber exactly...probably for the xpath search...